readme.md

```diff
@@ -3,6 +3,12 @@
 > ES2015 [`Object.assign()`](http://www.2ality.com/2014/01/object-assign.html) [ponyfill](https://ponyfill.com)
 
 
+## Use the built-in
```

Comment:
Should this be an own section before the install section though? I would place it after the API section.